src/LoggingFormats.jl

```diff
@@ -1,5 +1,37 @@
 module LoggingFormats
 
-greet() = print("Hello World!")
+using Logging
+
+function shorten_str(message, max_len)
+    suffix = "…"
+    if length(message) > max_len
+        message[1:min(end, max_len-length(suffix))] * suffix
+    else
+        message
+    end
+end
+
+# copied from https://github.com/JuliaLang/julia/blob/v1.5.4/stdlib/Logging/src/ConsoleLogger.jl and modified
+function make_log_truncated(max_var_len=5_000)
+    function log_truncated(io, args)
```

Comment:
I wonder if it would be nicer to implement this as a callable struct? Mostly because it takes some parameters it feels a bit "structy". E.g.
```julia
struct Truncated <: Function
    max_var_len::Int
end

function (tr::Truncated)(io, args)
    # ...
end
```
What do you think?

Reply:
I like it.

Reply:
Can we consider this resolved?